setup.go

```diff
@@ -82,6 +84,21 @@ func (e *Executor) readTaskfile() error {
 		Parent:     nil,
 		Optional:   false,
 	})
+
+	if e.Taskfile != nil && err != nil {
+		model := fuzzy.NewModel()
+		model.SetThreshold(1)
+		model.SetDepth(5)
+
+		var words []string
+		for taskName := range e.Taskfile.Tasks {
+			words = append(words, taskName)
+		}
+
+		model.Train(words)
+		e.fuzzyModel = model
+	}
```

Comment:
Extract this to its own function like the other below in this file. Could be named `setupFuzzyModel()`, for example.

Also, this function should be called from `Setup()` and not from `readTaskfile()`.